features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/store-new/source/src/app/components/Apis/Details/Credentials/Credentials.jsx

```diff
@@ -38,6 +38,7 @@ import InlineMessage from '../../../Shared/InlineMessage';
 import ExpressMode from './ExpressMode';
 import Subscription from '../../../../data/Subscription';
 import Alert from '../../../Shared/Alert';
```

Comment:
Can you please use the webpack path alias here, i:e `AppData`

Reply:
Fixed